tests/strategies/external/test_smart_meter.py

```diff
@@ -250,6 +312,27 @@ def test_delete_offer_aggregator(external_smart_meter: SmartMeterExternalStrateg
         assert return_value["command"] == "offer_delete"
         assert return_value["deleted_offers"] == [offer.id]
 
+    @staticmethod
+    def test_delete_offer_aggregator_deletes_offer_from_future_market(
```

Comment:
I think `test_delete_bid_aggregator_deletes_bid_from_future_market` is missing.

Reply:
Indeed, it is missing. I thought I added it, but apparently not. let me add it now.